src/theme/DocItem/index.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import {HtmlClassNameProvider} from '@docusaurus/theme-common';
+import {DocProvider} from '@docusaurus/theme-common/internal';
+import DocItemMetadata from '@theme/DocItem/Metadata';
+import DocItemLayout from '@theme/DocItem/Layout';
+import ReactMarkdown from 'react-markdown';
+import style from '@site/src/css/markdown-styles.module.css';
+import type {Props} from '@theme/DocItem';
+
+function SunsettedPage(message: string) {
+    return (
+        <div className={"sunset-message"}>
+            <ReactMarkdown className={style.reactMarkDown} >{"**" + message + "**"}</ReactMarkdown>
+        </div>
+    );
+}
+
+export default function DocItem(props: Props): JSX.Element {
+    const docHtmlClassName = `docs-doc-id-${props.content.metadata.id}`;
+    const MDXComponent = props.content;
+    const sunsettedPage = props.content?.frontMatter?.sunsetted === true;
+    const sunsettingMessage = "This project has been sunsetted and is no longer maintained. " + props.content?.frontMatter?.sunsettingMessage || "";
```

Comment:
Sunsetted is past tense and past participle. Sunsetted is also only grammatically correct when it is a noun or adjective. In this case, the word “sunset” is acting as a verb, meaning it doesn’t need “-ed”. Therefore, “sunsetted” is incorrect and “sunset” needs to be used here.

Reply:
Btw, **well aware** this is one of _those_ 🤓 reviews, just one to try and get credit ([Similar to Beg Bounties](https://www.troyhunt.com/beg-bounties/)), however with how large the project is and how many eyes are put on it each month, maintaining professional and correct customer-facing info is important. I genuinely couldn't care less if this is fixed or not, just thought I'd point it out. Regardless, I'm glad to see Waterfall being sunset, Velocity really is vastly superior and I'm glad this project has reached EOL. Thanks for all you do for the community! 😄

Reply:
"customer" 😆

Reply:
> "customer" 😆

What else do you want to call it? 💀